charts/kuberpult/templates/manifest-repo-export-service.yaml

```diff
@@ -235,6 +235,13 @@ spec:
         - name: KUBERPULT_RELEASE_VERSIONS_LIMIT
           value: "{{ .Values.git.releaseVersionsLimit }}"
         volumeMounts:
+        - name: repository
+          # The repository volume, an emptyDir, is mounted to the kp directory.
+          # The manifest-repo-export-service creates the repository inside this directory in /kp/repository.
+          # We mount the volume to the parent because kubernetes volumes belong to root.
+          # This way the container can create /kp/repository himself.
+          # DO NOT USE /kp TO STORE ANYTHING ELSE
```

Comment:
is it really accurate that we can't store anything else to /kp?
I thought the problem was that the export-service has to be the one to create the subdirectory.

Reply:
That is correct. However, this now means that the kp directory is the emptyDir volume. Meaning that if someone stores anything on this directory it will use the 30 Gi that we alocate for the git repository. This could lead to the repo service being restarted (happens when an empty dir is filled) without the repository itself beign 30Gi of size which could lead to a confusing debuging process.